packages/scripts/src/create-entrypoint.ts

```diff
@@ -136,21 +136,26 @@ async function main() {
     const topWindow = currentWindow.parent ?? currentWindow;
     const isMainEntrypoint = topWindow && currentWindow === topWindow;
     const options = new URLSearchParams(currentWindow.location.search);
+    const optionsObj = typeof $CODUX_TOPOLOGY !== undefined ? $CODUX_TOPOLOGY : {
+        publicPath: options.get('publicPath'),
+        ${FEATURE_QUERY_PARAM}: options.get('${FEATURE_QUERY_PARAM}'),
```

Comment:
this can generate invalid code. let's talk about another way to inject it. maybe via `globalThis.entryOptions`